### DISCUSSION TITLE
Publish your release images on Azure

### DISCUSSION THREAD

**Discussion by JayDoubleu** [actions/virtual-environments#4430] [Ideas]
It would be great if your releases would be publicly available on Azure.
A lot of projects will be forced to use private agentpools in ADO due to security constrains and currently we have to re-do entire release via custom made pipelines which costs money and time.

If your images for linux and windows would be published on Azure we could simply create scalesets out of them and have exact replica of MS hosted agent but in private and secure configuration which meets project needs.

I don't think this would be much work as those images are already out there*?

**Comment by miketimofeev** [comment]
Hi @JayDoubleu!
There was a discussion around that and I'm afraid it's not possible https://github.com/actions/virtual-environments/issues/176#issuecomment-620638942
